### PR TITLE
Netobserv 1.3 rel notes

### PR DESCRIPTION
**Pull request by skrthomas**
Release notes and bug fixes for the Network Observability 1.3 release. NOTE: builds will be broken until all feature content is merged on GA day, June 28, 2023. 

Version(s):

4.10+
Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
https://github.com/openshift/openshift-docs/assets/77501157/b1c2ef17-5d4a-4f58-b293-c46f892c08b5

Screen recording in lieu of a preview, since builds are broken.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [inline networking/network_observability/network-observability-operator-release-notes.adoc:18]: Should be 1.3.0

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [inline networking/network_observability/network-observability-operator-release-notes.adoc:27]: Misspelling of "For" and missing "see".  Should be:

For more information, see ...

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [inline networking/network_observability/network-observability-operator-release-notes.adoc:35]: Suggest:
Information about Network Observability Operator can now be included in the must-gather data for troubleshooting.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [inline networking/network_observability/network-observability-operator-release-notes.adoc:39]: Suggest:
Network Observability Operator can now run on an `amd64`, `ppc64le`, or `arm64` architecture.  Previously, it only ran on `x86_64`.

Comment (singh4jitendra) [inline networking/network_observability/network-observability-operator-release-notes.adoc:39]: I don;t see s390x here ?

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Addressed.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Addressed.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Addressed

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Sounds good to me. I updated the note.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Hmm good catch here. I removed this trailing space. I'm not sure I guess I maybe just accidentally spaced here and didn't realize it.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (memodi) [inline networking/network_observability/network-observability-operator-release-notes.adoc:43]:
```suggestion
* Previously, when the Operator was installed from the CLI, the `Role` and `RoleBinding` that are necessary for the Cluster Monitoring Operator to read the metrics were not installed as expected. The issue did not occur when the operator was installed from the web console. Now, either way of installing the Operator creates the required `Role and `RoleBinding`. (link:https://issues.redhat.com/browse/NETOBSERV-1003[*NETOBSERV-1003*])
```

**Review by memodi — CHANGES_REQUESTED** [review]
looks good overall, just a minor change.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (jotak) [inline networking/network_observability/network-observability-operator-release-notes.adoc:31]: typo "adds new a" => "adds a new"

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:43]: Addressed

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:39]: Addressed

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:58]: @jotak I agree with this...I can add another section header for Deprecated and removed features. Should relate the deprecation to the known issue at all? Or just have the deprecation mentioned in the new heading and then separately say, there is a known issue with the `HOST` setting?

Comment (jotak) [reply networking/network_observability/network-observability-operator-release-notes.adoc:65]: I think it's fine to relate the two of them, e.g. saying "HOST is deprecated blah blah AND it has a known issue etc."
Actually, whatever feels more natural to you, I think :)

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (jotak) [inline networking/network_observability/network-observability-operator-release-notes.adoc:46]:
```suggestion
The release of Network Observability Operator 1.3 deprecates the `spec.Loki.authToken` `HOST` setting. When using the Loki Operator, you must now only use the `FORWARD` setting.
```

(because people might still not use the loki operator, in which case nothing changes for them, they can use DISABLED)

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Good catch!

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (stleerh) [inline networking/network_observability/network-observability-operator-release-notes.adoc:63]: [NETOBSERV-1070](https://issues.redhat.com/browse/NETOBSERV-1070) is marked Closed in 1.3 so should this issue be listed here?  If so, the description doesn't tell me what problem will occur.  I don't know what to make of:

`flowlogs-pipeline servicemonitor` is not adapted to the TLS scheme.

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: @memodi @OlivierCazade Can you guys help me out with this Known Issue? I'm struggling to understand what context is missing that users will need to know to expect, aside from what is already stated. From our [slack](https://redhat-internal.slack.com/archives/C02939DP5L5/p1685460460174569) convo, it is my impression that there is no workaround.

Comment (memodi) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: @skrthomas Yes, if user sets it to `PROVIDED` there is no workaround, there was a follow up created [NETOBSERV-1087](https://issues.redhat.com/browse/NETOBSERV-1087) to fix that, may be that should be listed as known issue?

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: @memodi OK yeah, taking a look at this follow-up bug, _this_ seems like it is actually the known issue. I guess 1070 now has a patched fix but then 1087 is the remaining known issue?

Comment (memodi) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: of 3 modes, DISABLED, AUTO and PROVIDED for tls scheme ,  1070 was filed for issues with AUTO and PROVIDED mode and 1070 fixed AUTO mode. PROVIDED mode will be fixed with 1087.

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: will PROVIDED mode be fixed for 1.3? I'm guessing no, since I was asked to put this in the known issues.

Comment (memodi) [reply networking/network_observability/network-observability-operator-release-notes.adoc:63]: Correct,it won't be addressed for 1.3 timeframe, since @OlivierCazade had mentioned it involves crd changes.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (memodi) [reply networking/network_observability/network-observability-operator-release-notes.adoc:65]: how about notifying existing users who already may have HOST configured, may be this can serve as a notice to them until it's fully removed as part of [NETOBSERV-1084](https://issues.redhat.com/browse/NETOBSERV-1084) bug?

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:65]: That's a good suggestion @memodi. We also mention it in the Deprecated features area of the release notes that I just added.

Comment (memodi) [reply networking/network_observability/network-observability-operator-release-notes.adoc:65]: Sounds good. Thanks!

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (memodi) [inline networking/network_observability/network-observability-operator-release-notes.adoc:59]: @skrthomas NETOBSERV-1070 affects 1.2 release as well, we should probably mention under Bug fix. Something along the lines: Previously, when `processor.metrics.tls` is set to AUTO in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` is not adapted to the appropriate TLS scheme and metrics were not be visible under OCP console. Now the issue fixed for AUTO mode.

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:59]: tysm Mehul!  I think I misunderstood when we were talking about NETOBSERV-1070 previously, I guess I was thinking that only NETOBSERV-1087 was a known issue and I removed 1070. I will add 1070 back to the Bug Fix list.

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:59]: I think I previously had it as a Known Issue..

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (Amoghrd) [inline networking/network_observability/network-observability-operator-release-notes.adoc:60]: Now the issue `is` fixed for AUTO mode. Missing an is

Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Addressed

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (GroceryBoyJr) [inline networking/network_observability/network-observability-operator-release-notes.adoc:27]: This link is giving trouble in the Travis build - is it correct?

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (GroceryBoyJr) [inline networking/network_observability/network-observability-operator-release-notes.adoc:35]: This link is giving trouble in the Travis build. Is it correct?

**Review by GroceryBoyJr — COMMENTED** [review]
@skrthomas only two small nits please.

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc]: Adjusted per your wording suggestion. TY!

File: networking/network_observability/network-observability-operator-release-notes.adoc
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:27]: Its because this is a new feature and it is not yet merged. Basically, this is an asynchronous Operator and since our releases are outside of the normal OCP timeframe, I can't merge the features until GA day. The release notes will be the last thing I merge because all the links will be broken until all the features are merged.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by jotak** [comment]
/lgtm 
thanks @skrthomas !

**Pull request merged**
